またビルドエラー

@claude


> github-pages-demo@0.1.0 build
> next build

[baseline-browser-mapping] The data in this module is over two months old.  To ensure accurate Baseline data, please update: `npm i baseline-browser-mapping@latest -D`
⚠ No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

   ▲ Next.js 16.0.4 (Turbopack)

   Creating an optimized production build ...
[baseline-browser-mapping] The data in this module is over two months old.  To ensure accurate Baseline data, please update: `npm i baseline-browser-mapping@latest -D`
 ✓ Compiled successfully in 4.3s
   Running TypeScript ...
Failed to compile.

./app/components/BreakdownTab.tsx:127:13
Type error: Type '(value: number) => string' is not assignable to type 'Formatter<number, NameType>'.
  Types of parameters 'value' and 'value' are incompatible.
    Type 'number | undefined' is not assignable to type 'number'.
      Type 'undefined' is not assignable to type 'number'.

  125 |           </Pie>
  126 |           <Tooltip
> 127 |             formatter={(value: number) => formatCurrency(value)}
      |             ^
  128 |             contentStyle={{
  129 |               backgroundColor: 'white',
  130 |               border: '1px solid #e2e8f0',
Next.js build worker exited with code: 1 and signal: null
Error: Process completed with exit code 1.


これ以外にも、もうビルドエラーが起きないように全部チェックして直して。

--- a/app/components/BreakdownTab.tsx
+++ b/app/components/BreakdownTab.tsx
@@ -124,7 +124,9 @@ export function BreakdownTab({
             ))}
           </Pie>
           <Tooltip
-            formatter={(value: number) => formatCurrency(value)}
+            formatter={(value: number | undefined) =>
+              value !== undefined ? formatCurrency(value) : '¥0'
+            }
             contentStyle={{
               backgroundColor: 'white',
               border: '1px solid #e2e8f0',

